### PR TITLE
zh-translation: /news/releases/0.x/announcing-0.1/index.md #1519

### PR DESCRIPTION
**Pull request by gorda**
ref: [servicemesher#1519](https://github.com/servicemesher/istio-official-translation/issues/1519)

base: [Istio#arhive](https://github.com/istio/istio.io/blob/55d4514a0897a6920e98a2030b20c3d339fb6835/content/zh/blog/2017/0.1-announcement/index.md)

### CODE REVIEW

File: content/zh/news/releases/0.x/announcing-0.1/index.md [hunk 1]
```diff
@@ -15,89 +15,83 @@ aliases:
...
-Istio adds traffic management to microservices and creates a basis for value-add capabilities like security, monitoring, routing, connectivity management and policy.  The software is built using the battle-tested [Envoy](https://envoyproxy.github.io/envoy/) proxy from Lyft, and gives visibility and control over traffic *without requiring any changes to application code*. Istio gives CIOs a powerful tool to enforce security, policy and compliance requirements across the enterprise.
+Google、 IBM 和 Lyft 骄傲的宣布了 [Istio](/zh) 的首个公开版本，Istio 是一个提供一致的方式用于连接、加固、管理和监控微服务的开源项目。当前版本专注于支持 [Kubernetes](https://kubernetes.io/) 环境，我们计划在接下来的几个月添加诸如虚拟机和 Cloud Foundry 等环境的支持。
+Istio 为微服务添加了流量管理能力，同时为比如安全、监控、路由、连接管理和策略等附加能力打下了基础。此软件构建于来自 Lyft 的经过实战检验的 [Envoy](https://envoyproxy.github.io/envoy/) 代理之上，能在*无需改动任何应用代码*的情况下赋予对应用流量的可见性和控制能力。Istio 为 CIO 们提供了一个在企业内加强安全、策略和规范的强有力的工具。
 
-## Background
+## 背景
```
Comment (rootsongjc) [inline content/zh/news/releases/0.x/announcing-0.1/index.md]: 请增加锚点。

Comment (gorda) [reply content/zh/news/releases/0.x/announcing-0.1/index.md]: 已添加

File: content/zh/news/releases/0.x/announcing-0.1/index.md [hunk 2]
```diff
@@ -15,89 +15,83 @@ aliases:
...
+基于微服务模式编写可靠的、松耦合的、产品级的应用是有挑战的。随着巨型单体应用被分解为微服务，软件团队不得不面对将微服务集成进分布式系统的挑战：服务发现、负载均衡、故障容忍、端到端监测、动态路由，还有最重要的合规和安全。
 
-Inconsistent attempts at solving these challenges, cobbled together from libraries, scripts and Stack Overflow snippets leads to solutions that vary wildly across languages and runtimes, have poor observability characteristics and can often end up compromising security.
+层出不穷的方案尝试解决这些挑战，不同编程语言的解决方案差异巨大、缺少共性，这常常会危及到安全。
 
-One solution is to standardize implementations on a common RPC library like [gRPC](https://grpc.io), but this can be costly for organizations to adopt wholesale
-and leaves out brownfield applications which may be practically impossible to change. Operators need a flexible toolkit to make their microservices secure, compliant, trackable and highly available, and developers need the ability to experiment with different features in production, or deploy canary releases, without impacting the system as a whole.
+有一个解决方案是使用通用 RPC 库比如 [gRPC](https://grpc.io)，但是这在大规模适配时花销不菲，且可能在某些事实上无法更动的应用上留下棕色地带。运维人员需要一个灵活的工具来使他们的微服务变得安全、合规、可追踪和高可用，开发人员也需要这种能力来在产品环境实验不同的功能或者部署金丝雀版本而不影响系统的完整性。
```
Comment (vivian99-wu) [inline content/zh/news/releases/0.x/announcing-0.1/index.md]: 无法更动的应用
==》无法变更的应用，有没有更好一些？

File: content/zh/news/releases/0.x/announcing-0.1/index.md [hunk 3]
```diff
@@ -15,89 +15,83 @@ aliases:
...
+Istio 为微服务添加了流量管理能力，同时为比如安全、监控、路由、连接管理和策略等附加能力打下了基础。此软件构建于来自 Lyft 的经过实战检验的 [Envoy](https://envoyproxy.github.io/envoy/) 代理之上，能在*无需改动任何应用代码*的情况下赋予对应用流量的可见性和控制能力。Istio 为 CIO 们提供了一个在企业内加强安全、策略和规范的强有力的工具。
 
-## Background
+## 背景{#background}
 
-Writing reliable, loosely coupled, production-grade applications based on microservices can be challenging. As monolithic applications are decomposed into microservices, software teams have to worry about the challenges inherent in integrating services in distributed systems: they must account for service discovery, load balancing, fault tolerance, end-to-end monitoring, dynamic routing for feature experimentation, and perhaps most important of all, compliance and security.
+基于微服务模式编写可靠的、松耦合的、产品级的应用是有挑战的。随着巨型单体应用被分解为微服务，软件团队不得不面对将微服务集成进分布式系统的挑战：服务发现、负载均衡、故障容忍、端到端监测、动态路由，还有最重要的合规和安全。
 
-Inconsistent attempts at solving these challenges, cobbled together from libraries, scripts and Stack Overflow snippets leads to solutions that vary wildly across languages and runtimes, have poor observability characteristics and can often end up compromising security.
+层出不穷的方案尝试解决这些挑战，不同编程语言的解决方案差异巨大、缺少共性，这常常会危及到安全。
```
[the remark this answers is not in the document]
Comment (gorda) [reply content/zh/news/releases/0.x/announcing-0.1/index.md]: @vivian99-wu 已修改

**Pull request merged**